astropy/io/fits/tests/test_table.py

```diff
@@ -1074,8 +1074,8 @@ def test_multidimension_table_from_numpy_rec_columns(self):
         dx = fits.getdata(self.temp("test.fits"))
         assert data["x"].dtype == dx["x"].dtype
         assert data["y"].dtype == dx["y"].dtype
-        assert np.all(data["x"] == dx["x"]), "x: {} != {}".format(data["x"], dx["x"])
-        assert np.all(data["y"] == dx["y"]), "y: {} != {}".format(data["y"], dx["y"])
+        assert np.all(data["x"] == dx["x"]), f"x: {data['x']} != {dx['x']}"
```

Comment:
I think these extra clauses are from pre-pytest, where this was useful. Now pytest already shows the relevant data. But again probably out of scope to remove them...

Reply:
Agreed, but its more out of scope than I want to get right now.